nucliadb_vectors/src/data_point/ops_hnsw.rs

```diff
@@ -291,14 +291,13 @@ impl<'a, DR: DataRetriever> HnswOps<'a, DR> {
         let Some(entry_point) = hnsw.get_entry_point() else {
             return Neighbours::default();
         };
-        let mut crnt_layer = entry_point.layer;
+
         let mut neighbours = vec![(entry_point.node, 0.)];
-        while crnt_layer != 0 {
+        for crnt_layer in (0..=entry_point.layer).rev() {
```

Comment:
we were skipping layer 0

Reply:
hah, I thought it was on purpose. Should have asked!